README.md

```diff
@@ -0,0 +1,5 @@
+## Install
+
+### On a CentOS 7 system:
+
+    yum install libguestfs-tools autoconf virt-install virsh virt-sysprep virt-sparsify euca2ools python-requestbuilder
```

Comment:
euca2ools should pull in python-requestbuilder on its own.  Did you have to install it separately?